tools/mk/Makefile.node_prebuilt.targ

```diff
@@ -1,34 +0,0 @@
-# -*- mode: makefile -*-
```

Comment:
FWIW, we might not need to delete these files. Often the "tools/mk" dir has been a copy of *all* the eng.git Makefiles at the time. Until there is a specified real plan for consistency, I'm fine either way.